systemd/ocp-clusterid.service

```diff
@@ -5,6 +5,7 @@ After=kubelet.service
 [Service]
 Type=oneshot
 ExecStart=/usr/local/bin/ocp-clusterid.sh
+ExecCondition=/usr/local/bin/check-cloud-env.sh
```

Comment:
No need to change anything, but this is the only unit file where you put `ExecCondition` after `ExecStart`

Reply:
good catch, i'll make it consistent if i push any more changes :)